aviary/interface/graphical_input.py

```diff
@@ -212,12 +1253,12 @@ def create_bounds(center):
 
     return phase_info
 
-
 def estimate_total_range_trapezoidal(times, mach_numbers):
+    """Source: original Aviary graphical_input.py."""
     speed_of_sound = 343  # Speed of sound in meters per second
 
     # Convert times to seconds from minutes
-    times_sec = np.array(times) * 60
+    times_sec = np.array(times)
```

Comment:
Change or remove this part as it's no longer doing any conversion from minutes to seconds here

Reply:
Looking at that bit of code, I edited it so that the message that's shown to the user "Estimated range is xxx nautical miles" uses the same units for range as the units for altitude. Should I add logic to use nmi when altitude is in ft, or is it okay to stick with the same units as altitude?